### ISSUE TITLE
Xroot:  No connection from client after 300 seconds. Giving up.

### ISSUE THREAD

**Issue by alrossi** [dCache/dcache#6247]
This is a placeholder for a second issue we have seen with frequency at FNAL.   I am not sure it denotes a problem in dCache.

The typical access log for the problem we are seeing shows repeated attempts to log in with repeated redirects to the pool.

[door-access.txt](https://github.com/dCache/dcache/files/7514441/door-access.txt)

**Comment by paulmillar** [comment]
Did you have any luck getting the output from the client?

It might be helpful to hear the client's side of the story.

**Comment by alrossi** [comment]
The experiment experiencing this has not been contacted, but they have not opened a ticket yet either.

**Comment by alrossi** [comment]
I am attaching the access log for the door where a client is used that does not support TLS, but the pool is configured as STRICT TLS, but with TLS beginning after login (i.e., session).  The multiple opens plus the accumulating 'tried' with no triedrc are analogous to the log posted above.

I believe this issue is simply a configuration one, most probably compounded by the bug which was was corrected in https://rb.dcache.org/r/13229/

I am closing the issue for now, but if the problem represents itself at FNAL after our November downtime, we may reopen it

[xrootd-1094-fndcatemp2Domain.access.log](https://github.com/dCache/dcache/files/7563881/xrootd-1094-fndcatemp2Domain.access.log)
.

**Comment by alrossi** [comment]
I'll reopen this issue because the scenario I described does not fit what is on our pools currently.  It could be that a different problem is causing connection to the pool to fail.   I suspected TLS because Pool Manager does not know anything about a pool's TLS settings and so could keep selecting a pool which a client is incapable of talking to.  But we turned off TLS on our production pools back in October (for the moment).   So ....
